pkg/dmqnode-common.pkg

```diff
@@ -14,5 +14,6 @@ OPTS.update(
 # Replace existing args without rebinding
 ARGS[:] = [
         "README.rst=/usr/share/doc/" + VAR.shortname + "/README.rst",
-        "deploy/logrotate/dmqnode-logs=/etc/logrotate.d/"
+        "deploy/logrotate/dmqnode-logs=/etc/logrotate.d/",
+        "./deploy/systemd/dmq@.service=/lib/systemd/system/dmq@.service"
```

Comment:
Maybe just make it `deploy/systemd/dmq@.service` to be consistent with the other lines.